Userland/Services/LaunchServer/Launcher.cpp

```diff
@@ -349,4 +353,16 @@ bool Launcher::open_file_url(const URL& url)
 
     return open_with_user_preferences(m_file_handlers, extension, additional_parameters, default_handler);
 }
+
+bool Launcher::open_help_url(const URL& url)
+{
+    auto handler_optional = m_file_handlers.get("help");
+    if (!handler_optional.has_value())
+        return false;
+    auto& handler = handler_optional.value();
```

Comment:
Wait, so adding Help to the _file_ handlers is basically a hack to make this work...?

Reply:
Ahhhhhh this subsystem makes a lot more sense now. The `file` protocol does this and I was copying that which didn't really seem right but I figured it was that way for a reason. Gonna have to refiddle this a bit because it looks like Help is going to have to learn how to speak URL but the result should be a lot cleaner :)